README.rst

```diff
@@ -27,7 +27,9 @@ Patches
 Please submit patches via GitHub pull request on:
 https://github.com/labgrid-project/meta-labgrid
 
-Maintainer:  Enrico Jörns <e.joerns@pengutronix.de>
+Maintainers:
+- Enrico Jörns <e.joerns@pengutronix.de>
+- Czerwinski <r.czerwinski@pengutronix.de>
```

Comment:
Did you intentionally only use your surname?

Reply:
No, fixed.